contracts/SnappAuction.sol

```diff
@@ -111,8 +112,15 @@ contract SnappAuction is SnappBase {
         uint16 accountId = publicKeyToAccountMap(msg.sender);
         require(accountId <= AUCTION_RESERVED_ACCOUNTS, "Accout is not a reserved account");
 
+
         require(packedOrders.length % 26 == 0, "Each order should be packed in 26 bytes!");
         uint numOrders = packedOrders.length / 26;
+        uint32 currentBatchIndex = standingOrders[accountId].currentBatchIndex;
+        require(
```

Comment:
This constraint make me feel uneasy.

```
(batchIndex < MAX || numOrders == 0) &  batchIndex <= MAX
```
What scenario does this describe?

Reply:
(batchIndex < MAX || numOrders == 0) -> last order need to just to put 0 orders in there( annulating order)

&& batchIndex <= MAX -> makes sure that no overflow occurs